src/components/AnnotationStylePopup/AnnotationStylePopup.js

```diff
@@ -66,7 +66,9 @@ class AnnotationStylePopup extends React.Component {
           style={style}
           isFreeText={isFreeText}
           onStyleChange={this.handleStyleChange}
-          hideSlider={hideSlider}
+          hideOpacitySlider={hideSlider}
```

Comment:
added extra prop so we have a bit more control over what kind of slider to hide.
and inside style pop up:
hideSlider = hideOpacitySlider && .....